File: android/app/src/main/java/com/pizzadelivery/android/data/model/Store.kt
package com.pizzadelivery.android.data.model

import com.google.gson.annotations.SerializedName

data class Store(
    val id: String = "",
    val name: String = "",
    val address: StoreAddress = StoreAddress(),
    val location: Location = Location(),
    val hours: StoreHours? = null,
    @SerializedName("isOpen")
    val isOpen: Boolean = false,
    val rating: Double = 0.0,
    val phone: String = "",
    val image: String = "",
    val distance: Double? = null
)

data class StoreAddress(
    val street: String = "",
    val city: String = "",
    val state: String = "",
    val zipCode: String = "",
    val country: String = ""
)

data class Location(
    val lat: Double = 0.0,
    val lng: Double = 0.0
)

data class StoreHours(
    val monday: DayHours? = null,
    val tuesday: DayHours? = null,
    val wednesday: DayHours? = null,
    val thursday: DayHours? = null,
    val friday: DayHours? = null,
    val saturday: DayHours? = null,
    val sunday: DayHours? = null
)

data class DayHours(
    val open: String = "",
    val close: String = ""
)

File: ios/PizzaDelivery/Sources/Models/Store.swift
import Foundation

struct Store: Codable, Identifiable {
    let id: String
    var name: String
    var address: StoreAddress
    var location: Location
    var hours: StoreHours?
    var isOpen: Bool
    var rating: Double?
    var phone: String?
    var imageUrl: String?
    
    enum CodingKeys: String, CodingKey {
        case id, name, address, location, hours, isOpen, rating, phone, imageUrl
    }
}

struct StoreAddress: Codable {
    var street: String
    var city: String
    var state: String
    var zipCode: String
    
    var formatted: String {
        "\(street), \(city), \(state) \(zipCode)"
    }
}

struct Location: Codable {
    var lat: Double
    var lng: Double
}

struct StoreHours: Codable {
    var monday: DayHours?
    var tuesday: DayHours?
    var wednesday: DayHours?
    var thursday: DayHours?
    var friday: DayHours?
    var saturday: DayHours?
    var sunday: DayHours?
}

struct DayHours: Codable {
    var open: String
    var close: String
}
